systems/plants/@RigidBodyManipulator/contactConstraintDerivatives.m

```diff
@@ -1,13 +1,29 @@
-function [n, D, dn, dD] = contactConstraintDerivatives(obj, kinsol, idxA, idxB, xA, xB, normal, d, use_mex)
- compute_second_derivative = nargout > 2;
- 
-  if(use_mex)
+function [d, n, D, dn, dD] = contactConstraintDerivatives(obj, use_mex, normal, kinsol, idxA, idxB, xA, xB)
```

Comment:
I agree.  I think I did this in order to make it easier to test the call that only returns tangents before bringing surfaceTangentsmex back in.  Now it would be easier to just call surfaceTangentsmex directly.